app/src/main/res/values/strings.xml

```diff
@@ -116,17 +116,17 @@
 
     <string name="enter_group_name">Enter group name</string>
     <string name="groups">Groups</string>
-    <string name="noGroups">Click the "+" (plus) button to add groups first.\n\nGroups make things easier to find.</string>
+    <string name="noGroups">Click the "+" (plus) button to add groups for categorization first.</string>
```

Comment:
Why are you using ⊕ plus in the other menu but not here?

Reply:
I only appear to be peon of consistency.